doc/getting_started/toolchain_host.rst

```diff
@@ -0,0 +1,12 @@
+.. _host_toolchains:
```

Comment:
That's a very short page. Is it included or referenced anywhere?

Reply:
This is already the case:
https://builds.zephyrproject.org/zephyrproject-rtos/zephyr/16064/docs/getting_started/toolchain_host.html

Reply:
Neat! I wish the "details" link in the CI checks linked there instead of 404ing ;)